src/actions/index.js

```diff
@@ -0,0 +1,6 @@
+export const GET_BOOKLIST = 'GET_BOOKLIST'
+
```

Comment:
How necessary is this `const GET_BOOKLIST`? I haven't seen this in any actions in the lessons. I wonder if we should refactor it to just be the string? I'm curious if it will still work.

Reply:
I agree, I haven't seen this used as much. I'll do some research today & find out how necessary or advantageous it may be.